pkg/parca/parca.go

```diff
@@ -341,7 +343,10 @@ func Run(ctx context.Context, logger log.Logger, reg *prometheus.Registry, flags
 
 	var debuginfodClient debuginfo.DebuginfodClient = debuginfo.NopDebuginfodClient{}
 	if len(flags.Debuginfod.UpstreamServers) > 0 {
-		httpDebugInfoClient, err := debuginfo.NewHTTPDebuginfodClient(logger, flags.Debuginfod.UpstreamServers, flags.Debuginfod.HTTPRequestTimeout)
+		httpDebugInfoClient, err := debuginfo.NewHTTPDebuginfodClient(logger, flags.Debuginfod.UpstreamServers, &http.Client{
+			Transport: promconfig.NewUserAgentRoundTripper(fmt.Sprintf("parca-debugindod-%s", version), http.DefaultTransport),
```

Comment:
```suggestion
			Transport: promconfig.NewUserAgentRoundTripper(fmt.Sprintf("parca-debugindod/%s", version), http.DefaultTransport),
```

I belive using `/` as version separator is more common https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent

Reply:
fixed